joe.go

```diff
@@ -41,6 +41,17 @@ func availableFiles() (a []string, err error) {
 			availableGitignores = append(availableGitignores, strings.Replace(f.Name(), ".gitignore", "", 1))
 		}
 	}
+
+	files, err = ioutil.ReadDir(path.Join(dataPath, "Global"))
+	if err != nil {
+		return nil, err
+	}
+	for _, f := range files {
```

Comment:
This loop is repeating. Please abstract in a separate function to keep it consistent with DRY.